public/index.html

```diff
@@ -1,19 +1,18 @@
 <!DOCTYPE html>
```

Comment:
Why you made changes in this file? If you haven't done here anything, restore file to it's original from main.